wgpu/Cargo.toml

```diff
@@ -64,12 +68,11 @@ features = ["raw-window-handle"]
 # We want the wgpu-core Metal backend on macOS and iOS.
```

Comment:
comment here and on dx backends outdated now

Reply:
actually the entire block doesn't make sense anymore, no? wgc without a backend feature doesn't make any sense anyways

Reply:
This block is necessary to enable `wgpu-core` by default on MacOS & iOS. This lets `metal = ["wgc?/metal"]` enable the Metal backend only when `wgc` is active. E.g. when targeting WebGPU on Wasm, `wgpu-core` is not enabled, therefor the `metal` crate feature will not pull in any dependencies.

I find this horribly complicated, but unfortunately Cargo is very limited and these are the kind of workarounds we have to use to get around these issues.

Reply:
I tried to add some explanation to this block, let me know if it makes sense to you.